frontend/web/project/api.js

```diff
@@ -370,15 +370,18 @@ global.API = {
   setCookie(key, v) {
     try {
       if (!v) {
-        require('js-cookie').remove(key, {
-          domain: Project.cookieDomain,
-          path: '/',
-        })
-        require('js-cookie').remove(key, { path: '/' })
+        if (E2E) {
+          localStorage.removeItem(key, v)
+        } else {
+          require('js-cookie').remove(key, {
```

Comment:
Being curious: can `require('js-cookie')` be cached to a reusable var before the try statement enters?